pipeline.py

```diff
@@ -23,36 +24,33 @@
 MOCKED_STAGES = ["sdg", "train", "eval"]
 PIPELINE_FILE_NAME = "pipeline.yaml"
 IMPORTER_PIPELINE_FILE_NAME = "importer-pipeline.yaml"
-SDG_PIPELINE = "simple"
-IMAGE_PULL_SECRET = "redhat-et-ilab-botty-pull-secret"
+SDG_PIPELINE = "simple"  # FIXME: Is this a good default value?
+IMAGE_PULL_SECRET = (
+    "redhat-et-ilab-botty-pull-secret"  # FIXME: Is this a good default value?
+)
 STANDALONE_TEMPLATE_FILE_NAME = "standalone.tpl"
 GENERATED_STANDALONE_FILE_NAME = "standalone.py"
 DEFAULT_REPO_URL = "https://github.com/instructlab/taxonomy.git"
-KFP_MODEL_SERVER_CM = "sdg/kfp-model-server.yaml"
-
-# FIXME: This value is specific to ocp-beta-test.nerc.mghpcc.org cluster, `ilab` namespace. It is quite cumbersome to copypaste and remember the path every time in dev. This default value should go away once we reach feature freeze.
-BASE_MODEL = "s3://ilab-pipeline-b1d4c2b1-ab00-4e7f-b985-697bda3df385/instructlab-base-importer/648f36d0-e3f0-43b8-8adb-530576beb675/ilab-importer-op/model/granite-7b-starter"
 
 # eval args
-MMLU_TASKS_LIST = "mmlu_anatomy,mmlu_astronomy"
-FEW_SHOTS = 5
+FEW_SHOTS = 5  # FIXME: Is this a good default value?
 # BATCH_SIZE can also be an int, for example "8" is converted to an int in eval/final
-BATCH_SIZE = "auto"
-MAX_WORKERS = "auto"
-MERGE_SYSTEM_USER_MESSAGE = False
+BATCH_SIZE = "auto"  # FIXME: Is this a good default value?
+MAX_WORKERS = "auto"  # FIXME: Is this a good default value?
+MERGE_SYSTEM_USER_MESSAGE = False  # FIXME: Is this a good default value?
 
 # training args
-NUM_EPOCHS_PHASE_1 = 2
-NUM_EPOCHS_PHASE_2 = 2
-EFFECTIVE_BATCH_SIZE_PHASE_1 = 3840
-EFFECTIVE_BATCH_SIZE_PHASE_2 = 3840
-LEARNING_RATE_PHASE_1 = 1e-4
-LEARNING_RATE_PHASE_2 = 1e-4
-NUM_WARMUP_STEPS_PHASE_1 = 100
-NUM_WARMUP_STEPS_PHASE_2 = 100
-SAVE_SAMPLES = 0
-MAX_BATCH_LEN = 20000
-SEED = 42
+NUM_EPOCHS_PHASE_1 = 2  # FIXME: Is this a good default value?
+NUM_EPOCHS_PHASE_2 = 2  # FIXME: Is this a good default value?
+EFFECTIVE_BATCH_SIZE_PHASE_1 = 3840  # FIXME: Is this a good default value?
+EFFECTIVE_BATCH_SIZE_PHASE_2 = 3840  # FIXME: Is this a good default value?
+LEARNING_RATE_PHASE_1 = 1e-4  # FIXME: Is this a good default value?
+LEARNING_RATE_PHASE_2 = 1e-4  # FIXME: Is this a good default value?
+NUM_WARMUP_STEPS_PHASE_1 = 100  # FIXME: Is this a good default value?
+NUM_WARMUP_STEPS_PHASE_2 = 100  # FIXME: Is this a good default value?
+SAVE_SAMPLES = 0  # FIXME: Is this a good default value?
+MAX_BATCH_LEN = 20000  # FIXME: Is this a good default value?
+SEED = 42  # FIXME: Should we default to this? What is a good default value? 0?
```

Comment:
I don’t believe we need all the FIXMEs here. The defaults are sourced from instructlab/instructlab and are identical, so they can be trusted in my opinion.

Reply:
Also, I feel like this is unrelated to the main change of running the pipeline from the CLI. Should we treat this separately?

Reply:
Maybe we should do this separately. I don't know which some from instructlab/instructlab and which don't, that's why I'm asking in the PR description about this.

Reply:
I suggest breaking this into two PRs. Completing the pipeline run can be prioritized and done more quickly, while the review and adjustments for all the knobs may take more time.

Just to clarify, I checked the values, and they are sourced from InstructLab. I believe they are fine as they are.

Reply:
Removed from the PR.